### DISCUSSION TITLE
Docker Redash - inside the Application Gateway Azure

### DISCUSSION THREAD

**Discussion by JackStuntman** [getredash/redash#6101] [General]
Hi all
I'm new to using Redash and also Redash inside a docker compose.
I am writing to you for a question.
I'm developing an infrastructure with a VM where a docker with Redash is installed.
I configured a Backend setting on the application gateway to expose the input port (nginx) to redash (port 91)
If I try to connect on "private ip":91 it works
while if I write "public ip"/redash/ (so without port) as configured by appGw
tries to load the Redash but the components (assett, js ...) do not load them because it takes the root as path.
is there any configuration to apply on docker to take the correct path and not the root?
Do you have any tips to recommend me?
I hope I was clear
Thank you

**Comment by Akanshu9990** [comment]
hey!
You got any solution for this? as I'm also stuck here
